ONSdigital/eq-author-app
# Reordering answers

### What is the context of this PR?
Adds the ability to reorder answers by using the up and down arrows as per the design in survey introduction for collapsibles.

This also bumps the react, react-dom and enzyme versions so we can use hooks! 🎉 

### How to review 
1. Ensure you can re-arrange answers
2. Ensure the animation is smooth
3. Ensure that the re-arrangement is persisted
